src/reynir_correct/main.py

```diff
@@ -341,17 +343,43 @@ def sentence_stream() -> Iterator[List[CorrectToken]]:
             )
             for ann in a
         ]
+        if args.text:
+            arev = a
```

Comment:
This actually creates a second reference to the same list; the list is not copied. Better:

`arev = sorted(a, key=...)`

Reply:
Done